### PR TITLE
Team Fusion - First Blog Post (RGSoC 2017)

### PR DESCRIPTION
**Pull request by Janakshi28**
These files are to be used for the first blog post of Team Fusion, which is to be published on 2017-07-20. This contains 12 image files, which includes jpgs and gifs. All of the jpgs are having a width of 600px.

### CODE REVIEW

**Comment by alicetragedy** [comment]
Hi @Janakshi28, if that's possible, please add your blog post and the images to the same pull request (e.g. when you create and commit your blog post, you should be able to update this PR with the post); this makes it easier for us to review. Otherwise, trying out your blog post locally will show me broken images :) 
I hope that's understandable, but if it's not, and if you have any questions about how the pull request stuff works, don't hesitate to ping me!

**Comment by Janakshi28** [comment]
Hi @alicetragedy, Oh sorry, we'll do that.. We thought we are supposed to get the images merged first.. We'll add both the images and the blog post to the same PR.

**Comment by Janakshi28** [comment]
Hi, we have added the blog post file and made few changes in the content too. 🙂

Request for review: @alicetragedy

File: blog/_posts/2017-07-20-meet-team-fusion.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+![Team Fusion Pair](/img/blog/2017/2017-07-20-jana-and-kal.gif)
+<div class="image-credits"><b>The two of us (Image : Janakshi's Tab)</b></div>
+
+We are _Janakshi_ and _Kalpani_ from **Team Fusion**, working on project **OpenDF**. We are not just any team, but the first ever Sri Lankan team, to be a part of RGSoC Journey and the one and only Sri Lankan team from RGSoC 2017! 😌🎉 We feel so lucky to be a part of RGSoC family and we thank Rails Girls Summer of Code team for giving us this golden opportunity.. 😇
+
+## So who are we?
+
+We are two young ladies who love to code. But, that's not enough for an introduction.. So this is who we really are. 👇
+
+[__Jankashi :__](https://lk.linkedin.com/in/janakshidulanga) Janakshi Dulanga (22 years) is from Kegalle, Sri Lanka. She’s someone who loves to do lots of things, that's right, a multi-tasker. Cooking comes first, then wandering about stars and music, and guess what? She loves Programming too.. She had to choose them all! She’s currently a software engineering undergraduate, from APIIT Sri Lanka, who's waiting for her final degree results, and hoping to become a Software Engineer who can cook! 🔭💻👩‍🍳
```
Comment (alicetragedy) [inline blog/_posts/2017-07-20-meet-team-fusion.md]: minor typo here, it should be “Janakshi” :)

**Review by alicetragedy — APPROVED** [review]
looks good @Janakshi28! there's just a minor typo, I commented on it but can also fix this myself later. thanks for submitting your blog post on time!

**Comment by Janakshi28** [comment]
Hi @alicetragedy, Sorry about that. I fixed the typo.. 😁 and Thank you :)

**Review by alicetragedy — APPROVED** [review]

**Pull request merged**